.github/workflows/acctest.yml

```diff
@@ -68,10 +68,25 @@ jobs:
 
         METAL_AUTH_TOKEN: ${{ secrets.PACKET_AUTH_TOKEN }}
       run: |
-        go test -v -cover -parallel 4 -timeout 120m ./metal
+        go test -v -json -coverprofile coverage.txt -covermode=atomic -parallel 4 -timeout 180m ./metal > test.json
```

Comment:
Do coverage.txt and test.json have to be unique names for the TF 0.13 test and 1.0?

Reply:
Can we increase the parallelization?